aten/src/ATen/cpu/vec/vec256/vec256.h

```diff
@@ -149,16 +149,22 @@ inline convert_to_int_of_same_size<float>(const Vectorized<float> &src) {
   return _mm256_cvttps_epi32(src);
 }
 
-// Only works for inputs in the range: [-2^51, 2^51]
 // From: https://stackoverflow.com/a/41148578
```

Comment:
Would be good to update the link to the stack overflow answer here.

In that same thread, there is also an, apparently, faster implementation at https://stackoverflow.com/a/41148578/5280578 (haven't benchmarked).

Reply:
Do you mean this implementation which uses 6 instructions? The implementation I used in this PR taking 5 instructions, should be better?
```c++
__m128d int64_to_double_full(__m128i x){
    __m128i xH = _mm_srai_epi32(x, 16);
    xH = _mm_blend_epi16(xH, _mm_setzero_si128(), 0x33);
    xH = _mm_add_epi64(xH, _mm_castpd_si128(_mm_set1_pd(442721857769029238784.)));              //  3*2^67
    __m128i xL = _mm_blend_epi16(x, _mm_castpd_si128(_mm_set1_pd(0x0010000000000000)), 0x88);   //  2^52
    __m128d f = _mm_sub_pd(_mm_castsi128_pd(xH), _mm_set1_pd(442726361368656609280.));          //  3*2^67 + 2^52
    return _mm_add_pd(f, _mm_castsi128_pd(xL));
}
```